src/trait-bounds.md

```diff
@@ -73,6 +73,26 @@ fn name_figure<U: Shape>(
 }
 ```
 
+Bounds that don't use the item's parameters or [higher-ranked lifetimes] are
+checked when the item is defined. It is an error for such a bound to be false.
+
+[`Copy`], [`Clone`], and [`Sized`] bounds are also checked for certain generic types when using the item, even if the use does not provide a concrete type.
+It is an error to have `Copy` or `Clone` as a bound on a mutable reference, [trait object], or [slice][arrays].
```

Comment:
I think we've added a slice chapter since this was written, can you update the link?

```suggestion
It is an error to have `Copy` or `Clone` as a bound on a mutable reference, [trait object], or [slice].
```

I think it would point to:

```
[slice]: types/slice.md

Reply:
Thanks! Updated in the upstream branch (for #1060) and rebased.